client/modules/IDE/components/Preferences.jsx

```diff
@@ -229,6 +230,25 @@ class Preferences extends React.Component {
               </div>
             </div>
           </TabPanel>
+          <TabPanel>
+            <div className="preference">
+              <h4 className="preference__title">Privacy Settings</h4>
+              <div className="preference__options">
+                <input
+                  type="checkbox"
+                  onChange={(event) => {
+                    this.props.setIsPrivate(event.target.checked);
+                  }}
+                  aria-label="privacy output on"
+                  name="privacy output"
+                  id="privacy-output-on"
+                  value="On"
+                  checked={(this.props.isPrivate)}
```

Comment:
Unnecessary `()` brackets, consider removing them or alternatively wrapping in a `Boolean()` call if it's really necessary.